src/desktop/apps/artwork/maybe_new_artwork_page.ts

```diff
@@ -0,0 +1,16 @@
+const { NEW_ARTWORK_PAGE_THRESHOLD } = require("desktop/config.coffee")
+
+module.exports = (_req, res, next) => {
+  // When not set, we serve the existing artist page.
+  if (!NEW_ARTWORK_PAGE_THRESHOLD) {
+    return next()
+  }
+
+  const { NEW_ARTWORK_PAGE } = res.locals.sd
+  if (parseInt(NEW_ARTWORK_PAGE, 10) < NEW_ARTWORK_PAGE_THRESHOLD) {
```

Comment:
Should we store this random assignment somewhere and reuse it. Unless I'm missing something - the way it's written right now - one can get one page on the first load and an old one in reload, no?

Reply:
`NEW_ARTWORK_PAGE` is already stored, that's the normal 'random or your existing cookie' value. So everyone gets set from 0-9 randomly (with equal prob.), and that doesn't change.

Then, we configure the threshold, which is the cutoff of who sees it.

Reply:
Oh... Sorry. 3 hours of sleep today:) I read `parseInt` as random. Thanks for chewing it up for me here...